dev-python/influxdb/influxdb-5.3.0-r1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
One blank line after EAPI as per the [skeleton](https://gitweb.gentoo.org/repo/gentoo.git/tree/skel.ebuild) ebuild (or see the proxy-maint style [guide](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide)).

Reply:
Done (by way of explanation, it was like this before; I just copied the old ebuild).